sdk/core/Azure.Core/README.md

```diff
@@ -82,9 +83,25 @@ SecretClient client = new SecretClient(new Uri("http://example.com"), new Defaul
 
 More on client configuration in [client configuration samples](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Configuration.md)
 
+### Customzing Request Using `RequestContext`
+
+Besides general configuration of _serice client_ through `ClientOptions`, you can customize each individual request sent by _service client_
+using `RequestContext`.
+
+_Service clients_ have methods that can be used to call Azure services. We refer to these client methods _service methods_. You can
+pass `RequestContext` to each _service method_ as a parameter.
+
+```C# Snippet:SetRequestContext
+Response response = await client.GetPetAsync("snoopy", new RequestContext()
+{
+    ErrorOptions = ErrorOptions.Default
```

Comment:
I think it's unrealistic that we'd create a new instance of `RequestContext` to set a property to its default value -- could we instead create a new RequestContext and call `AddClassifier(404, isError: false)` on it before passing it to the method?

Reply:
Replaced with a classifier 404 case.